fixtures/watcher/remount.html

```diff
@@ -1,75 +1,95 @@
-<!doctype html>
+<!DOCTYPE html>
```

Comment:
Apologies in advance, looks like Prettier did some formatting, I can revert if needed